src/styles/blocks/_card.scss

```diff
@@ -0,0 +1,82 @@
+.card {
+  width: 200px;
+  background-color: $white-color;
+  border-radius: 5px;
+  border: 1px solid $light-grey;
+  padding: 32px 16px 16px;
+
+  &__image {
+    display: block;
+    margin: 0 auto;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__product {
+    color: $text-color-first;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin: 40px 0 0;
+  }
+
+  &__product-code {
+    color: $text-color-second;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    margin: 4px 0 0;
+  }
+
+  &__stars {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin: 16px 0 0;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-top: 24px;
+    margin-bottom: 16px;
+
+    &-label {
+      color: $text-color-second;
+      font-size: 12px;
+      font-weight: 400;
+      line-height: 18px;
+      margin: 0;
+    }
+
+    &-value {
+      color: $text-color-first;
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+      margin: 0;
+    }
+  }
+
+  &__button {
+    display: block;
+    border-radius: 5px;
+    line-height: 40px;
+    height: 40px;
+    width: 166px;
+    text-align: center;
+    border: $blue-color;
```

Comment:
The border property is missing a size and style. It should look like 'border: 1px solid $blue-color'.